doc/developer/chips/8.rst

```diff
@@ -208,14 +209,15 @@ Default vs strict mode
 To address the desire for both fast prototyping and enforced error handling,
 the compiler will run checks in one of two modes, based on a flag.
 
-1. **Default.** If a call to a throwing procedure is not enclosed by a ``try``
-   or ``try!``, the program will compile with a warning and the call will
-   ``halt()`` on an error.
+If a call to a throwing procedure is not enclosed by a ``try`` or ``try!``: 
 
-2. **Strict.** If a call to a throwing procedure is not enclosed by a ``try``
-   or ``try!``, the compiler will raise an error.
+1. **Default.**: the program will compile with a warning. An error will
+   be propagated by the calling procedure if it ``throws``, and will
+   ``halt()`` if it does not throw.
```

Comment:
That's a fair point, if the developer wants warnings then they should compile in Strict mode. I'll update this to reflect that.